test/interleaver/do_interleave_test.exs

```diff
@@ -29,7 +29,6 @@ defmodule Membrane.DdInterleaveTest do
       cases = [
         {1, <<1, 90, 2, 100, 3, 110, 4, 120>>},
         {2, <<1, 2, 90, 100, 3, 4, 110, 120>>},
-        {3, <<1, 2, 3, 90, 100, 110, 4, 120>>},
```

Comment:
What happened here? What is the reason for this change?

Reply:
in this test I take 4 bytes_per_channel for each checked case, so I would have to create another test for the case with sample_size 3, which I thought was unnecessary (previously 'interleave' didn't check if bytes_per_channel were multiple of sample_size as it was ensured by AudioInterleave element, and 'interleave' just interleaved shorter chunks at the end).

Reply:
So I will add another test for sample size 3 so that the test are complete.